projects/ad9081/src/app_clock.c

```diff
@@ -128,7 +115,7 @@ int32_t app_clock_init(struct clk clk[NUM_APP_CLKS])
 		.in_buf_mode = {0x07, 0x07, 0x00, 0x00, 0x15},
 		.gpi_ctrl = {0x00, 0x00, 0x00, 0x00},
 		.gpo_ctrl = {0x37, 0x33, 0x00, 0x00},
-		.num_channels = NUM_APP_CLKS,
+		.num_channels = 6,
```

Comment:
make sure to use the channel number define here as well

Reply:
This is outdated code, isn't it?

Reply:
yeah, sorry, i must've done something wrong that i saw and commented outdated code